mobile/src/main/java/org/openhab/habdroid/util/Util.java

```diff
@@ -185,6 +167,32 @@ public static void setActivityTheme(@NonNull final Activity activity, String the
         return R.style.HABDroid_Light;
     }
 
+    public static void checkFullscreen(Activity activity) {
```

Comment:
Why not move those as protected methods to `BaseActivity`?

Reply:
Done